mapdata/WarcraftLegacies/UpgradeData/Skin/842089050.json

```
@@ -43,7 +43,7 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Allows Silithid Colossi to use Cleaving Attack, which causes \u003CZB20,DataA1,%\u003E% of their attack damage to strike through to nearby enemies."
+      "Value": "Allows Super Major to use Cleaving Attack, which causes \u003CZB20,DataA1,%\u003E% of their attack damage to strike through to nearby enemies."
```

Not an expert on insect words but I think the plural is probably this:
```suggestion
      "Value": "Allows Super Majors to use Cleaving Attack, which causes \u003CZB20,DataA1,%\u003E% of their attack damage to strike through to nearby enemies."
```